src/Symfony/Component/BrowserKit/HttpBrowser.php

```diff
@@ -75,7 +75,7 @@ private function getBodyAndExtraHeaders(Request $request): array
         $fields = $request->getParameters();
 
         if ($uploadedFiles = $this->getUploadedFiles($request->getFiles())) {
-            $part = new FormDataPart($uploadedFiles);
+            $part = new FormDataPart(array_merge($uploadedFiles, $fields));
```

Comment:
Here, the order looks wrong. We should probably swap $fields and $uploadedFiles (to be consistent with the previous behavior). Also, can you add a test to avoid any regression in the future.

Reply:
Good spot, I've fixed this and created a second test as suggested. BTW Enjoyed watching the pull request being reviewed online.